Nighttime-Operations/Auxiliary-Telescope/AT-Shutdown/Shutdown.rst

```diff
@@ -8,10 +8,117 @@
 Shutdown
 #######################
 
-Under construction.
+Stopping Scheduler
+^^^^^^^^^^^^^^^^^^
+
+The Scheduler automatically adds new targets to the queue until the start of nautical morning twilight.
+If there are no targets, the Scheduler will stop adding new ones.
+If you need to stop the Scheduler manually, you can use the ``auxtel/scheduler/stop.py`` script from ScriptQueue without any additional configurations.
+
+See more detailed descriptions on `Scheduler <https://obs-ops.lsst.io/Nighttime-Operations/Scheduler/night-time-operations/index.html>`__.
+
+
+Shutdown AuxTel
+^^^^^^^^^^^^^^^
+
+After stopping scheduler, you can shutdown AuxTel using ``axutel/shutdown.py`` script from ScriptQueue. 
+``shutdown.py`` script does not any configurations. This script includes:
+
+**1. Closing the M1 cover.**
+: First, disable ATAOS corrections. Then, close the M1 cover and vent gates.  
+If mirror is opened, ``shutdown`` script will not able to close the dome slit. 
+
+**2. Closing the Dome Shutter.**
+: Closing the dome shutter and dropout door if it is opened.  
```

Comment:
Change "closing" to "closes"

Reply:
Do you mean "Closing" on line 32 only? I also wonder if it is "close" or "closes".